common/ReduxChannel.js

```diff
@@ -0,0 +1,87 @@
+const DEFAULT_OPTIONS = {
+  timeout: 20000,
```

Comment:
Nit: i assume this is in ms, but it'd be nice to have a friendly comment w/ times with a user friendly value. 
For example: `timeout: 20000, // 20 seconds`